tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItLinkedBrokerParcelable.java

```diff
@@ -255,6 +255,16 @@ public boolean equals(Object o) {
         return linkedLogin.userId.equals(that.linkedLogin.userId);
     }
 
+    public boolean equalsAccounts(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        TradeItLinkedBrokerParcelable that = (TradeItLinkedBrokerParcelable) o;
+
+        return linkedLogin.userId.equals(that.linkedLogin.userId) &&
```

Comment:
@mitochondrion the userToken will need to be read from the keychain any time this is done. That's a very heavy operation to do every 30 seconds.